### DISCUSSION TITLE
OpenDTU: Is it possible to customize Live View page?

### DISCUSSION THREAD

**Discussion by JackDesere** [tbnobody/OpenDTU#1638] [General]
Is it possible to customize Live View page?
Now you see: Total Yield Total  |  Total Yield Day  |  Total Power.
Total Yield Total = total value from the beginning. Can I change the start date to 1 Januari for example?
Is it possible to add "Total week"   |   "Total Month" value?

**Comment by NintendoTito** [comment]
The data that you see on the start page is data that is saved by the inverter. OpenDTU only reads this data from the inverter and does NOT calculate it itself.

Therefore, no specific time periods can be set at this point. The only thing that goes in this direction: You can store a yield offset in the OpenDTU "Inverter" settings. However, this is usually used to "zero out" a used inverter for your own use.

The wish you have expressed therefore fits more in the direction of an evaluation and/or visualisation. You can do this, for example, by sending the data to Home Assistant via MQTT.

-----------------------------------------------------------------------------

Die Daten, welche du auf der Startseite siehst, sind Daten welche vom Wechselrichter gespeichert werden. OpenDTU liest diese nur vom WR und berechnet diese NICHT selber.

Daher können an der Stelle keine bestimmten Zeiträume eingestellt werden. Das einzige was in die Richtung geht: In den Einstellungen von OpenDTU "Wechselrichter" kannst du einen Ertragsversatz hinterlegen. Dieser dient aber üblicherweise eher dazu, einen gebraucht gekauften WR für die eigene Nutzung "auszunullen".

Der von Dir geäußerte Wunsch passt somit wieder eher in die Richtung einer Auswertung und/oder Visualisierung. Dies kannst du z. B. über die Versendung der Daten per MQTT an Home Assistant vornehmen.
